ios/AmplitudeReactNative.swift

```diff
@@ -247,12 +247,39 @@ class ReactNative: NSObject {
         Amplitude.instance(withName: instanceName).setServerZone(ampServerZone, updateServerUrl: updateServerUrl)
         resolve(true)
     }
-    
+
+    @objc
+    func setEventUploadMaxBatchSize(_ instanceName: String,
+                    eventUploadMaxBatchSize: Int32,
+                    resolver resolve: RCTPromiseResolveBlock,
+                    rejecter reject: RCTPromiseRejectBlock) -> Void {
+        Amplitude.instance(withName: instanceName).setEventUploadMaxBatchSize(eventUploadMaxBatchSize);
+        resolve(true)
+    }
+
+    @objc
+    func setEventUploadPeriodMillis(_ instanceName: String,
+                    eventUploadPeriodMillis: Int32,
+                    resolver resolve: RCTPromiseResolveBlock,
+                    rejecter reject: RCTPromiseRejectBlock) -> Void {
+        Amplitude.instance(withName: instanceName).eventUploadPeriodSeconds = eventUploadPeriodMillis;
```

Comment:
Do we need to adjust milliseconds to seconds here?

Reply:
Yes, you are right. Will fix this.